code/game/objects/items/devices/scanners.dm

```diff
@@ -57,9 +57,6 @@ REAGENT SCANNER
 			var/mutable_appearance/MA = new(O)
 			MA.alpha = 128
 			MA.dir = O.dir
-			if(MA.layer < TURF_LAYER)
```

Comment:
Stuff in this file as well seems to be incorrectly merged

Reply:
Rebased branch based on an updated one with the same changes (I think)